api-reference/v1.0/api/directoryrole_delete_member.md

```diff
@@ -0,0 +1,67 @@
+# Remove directory role member
+
+Use this API to create a new directory role member.
```

Comment:
Same issues in this v1.0 topic need to be fixed as identified on the beta topic.